File: api-examples/cat/count/python/attribute_search_spatial_circle.py
import requests

url = "https://<catalogue-server-ip>/catalogue/v1/count?attribute-name=(tags)&attribute-value=((traffic))&lat=18.4&lon=73.9&radius=200000"

payload = {}
headers= {}

response = requests.request("GET", url, headers=headers, data = payload)

print(response.text.encode('utf8'))

File: api-examples/cat/count/js/attribute_search_spatial_circle.js
var settings = {
  "url": "https://<catalogue-server-ip>/catalogue/v1/count?attribute-name=(tags)&attribute-value=((traffic))&lat=18.4&lon=73.9&radius=200000",
  "method": "GET",
  "timeout": 0,
};

$.ajax(settings).done(function (response) {
  console.log(response);
});
